cmd/cigarToBed/cigarToBed.go

```diff
@@ -0,0 +1,198 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/align"
+	"github.com/vertgenlab/gonomics/bed" //TODO: raven added this line for cigarToBed
+	"github.com/vertgenlab/gonomics/fasta"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/genomeGraph"
+	"log"
+	"os"      //raven added this line for MSA Fasta output
+	"strings" //raven added this line for CountSeqIdx
+)
+
+//raven did not put this helper function into the globalAlignment function because it is used twice within the globalAlignment function
```

Comment:
By globalAlignment function, are you referring to the function on line 60 or a function in a different file?